WordPress/src/main/java/org/wordpress/android/ui/TextInputDialogFragment.java

```diff
@@ -78,9 +79,14 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
             hintView.setVisibility(View.GONE);
         }
 
-        if (!isMultiline) {
-            editText.setMaxLines(1);
+        if (isMultiline) {
+            editText.setSingleLine(false);
+            editText.setMaxLines(10);
+            editText.setHorizontallyScrolling(false);
```

Comment:
I believe horizontally scrolling defaults to `false` so this probably isn't necessary.

Reply:
You are right. I test it and it works well.